Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -51,6 +66,37 @@ namespace AWSGameLift
         }
     }
 
+    GameLiftServerProcessDesc AWSGameLiftServerManager::BuildGameLiftServerProcessDesc()
+    {
+        GameLiftServerProcessDesc serverProcessDesc;
+        AZ::IO::FileIOBase* fileIO = AZ::IO::FileIOBase::GetDirectInstance();
+        if (fileIO)
+        {
+            const char pathToLogFolder[] = "@log@/";
+            char resolvedPath[AZ_MAX_PATH_LEN];
+            if (fileIO->ResolvePath(pathToLogFolder, resolvedPath, AZ_ARRAY_SIZE(resolvedPath)))

```

Comment:
Minor: Why do we need to resolve the path? What happens if the logs have not been generated yet and so the path does not exist?

Reply:
It is copied from original place without any change.